db/sql/V4.7__insert_klab2_schema.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN TRANSACTION;
+
+INSERT INTO ref_cluster (name, disaster_recovery, on_prem, on_hardware, is_default) VALUES
+  ('klab2', false, true, false, false)
+ON CONFLICT (name) DO NOTHING;
+END TRANSACTION;
```

Comment:
why end TRANSACTION here and start a new one in a same migration script?

Reply:
Woops. Had this for debugging. Removed this extraneous code.